demisto_sdk/commands/validate/validate_manager.py

```diff
@@ -1015,6 +1015,9 @@ def validate_beta_integration(self, structure_validator, pack_error_ignore_list)
         return integration_validator.is_valid_beta_integration()
 
     def validate_image(self, file_path, pack_error_ignore_list):
+        pack_name = get_pack_name(file_path)
+        if pack_name == 'NonSupported':
+            return True
```

Comment:
Why is thus suddenly needed?

Reply:
Cause it validates the NonSupported for Qradar_5_4_9 and it causes a lot of problem in the regular build. 
We don't need to validate this pack at all in my opinion

Reply:
The problem exists without your changes?

Reply:
It is still happening for some reason at the build - because it runs `validate all` probably
If I do update the image, and I need to update also the size to be compatible, it kinda deletes the old file and add the image as new - which fails the validation completely as the file it looking for doesn't exists anymore. 
WDYT we should do?
Open a PR to fix only the naming and force merge or Open a PR fix the entire image and force? 
Or ignore this pack at all from our entire validation